pages/_app.js

```diff
@@ -10,14 +10,13 @@ import Layout from "./layout";
 const fetcher = (url) => fetch(url).then((response) => response.json());
 
 export default function App({ Component, pageProps }) {
-  // This is only meant as a temporary solution to keep the App working as long as backend create, update and delete are not implemented (next US)
-  const [transactions, setTransactions] = useState([]);
   const router = useRouter();
 
-  const { data: initialTransactions, isLoading } = useSWR(
-    `/api/transactions`,
-    fetcher
-  );
+  const {
+    data: initialTransactions,
```

Comment:
is there a reason to call them initialTransactions? They will always be the current transactions you get from the backend, so the name can be misleading. Please find a better one.